chrome/content/zotero-better-bibtex/schomd.coffee

```diff
@@ -117,8 +191,10 @@ Zotero.BetterBibTeX.schomd.citation = (citekeys, {style, libraryID} = {}) ->
 
   return '' if itemIDs.length == 0
 
-  url = "http://www.zotero.org/styles/#{style ? 'apa'}"
-  style = Zotero.Styles.get(url)
+  urlstyle = "#{style ? 'apa'}"
```

Comment:
Looking very good in general, so these are really just nitpicks, sorry: wouldn't `style ||= 'apa'` achieve the same (it'd be a little faster)